tests/testParquet.py

```diff
@@ -0,0 +1,149 @@
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.    See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+
+from __future__ import print_function
+
+import os, shutil
+import unittest
+import tempfile
+
+import pandas as pd
+import pyarrow as pa
+import pyarrow.parquet as pq
+
+from pandas.util.testing import assert_frame_equal
+
+import lsst.utils.tests
+
+from lsst.qa.explorer.parquetTable import ParquetTable, MultilevelParquetTable
+
+ROOT = os.path.abspath(os.path.dirname(__file__))
+
+def setup_module(module):
+    lsst.utils.tests.init()
+
+class ParquetTableTestCase(unittest.TestCase):
+    """Test case for ParquetTable
+    """
+    testFilename = 'simple_test.parq'
+
+    def setUp(self):
+        self.df = pq.read_table(os.path.join(ROOT, self.testFilename)).to_pandas()
+        self.tempDir = tempfile.gettempdir()
+        self.filename = os.path.join(self.tempDir, self.testFilename)
+
+        table = pa.Table.from_pandas(self.df)
+        pq.write_table(table, self.filename, compression='none')
+        self.parq, self.dfParq = self.getParq()
+
+    def tearDown(self):
+        del self.df
+        del self.parq
+        os.remove(self.filename)
+
+    def getParq(self):
+        return ParquetTable(self.filename), ParquetTable(dataFrame=self.df)
+
+    def testRoundTrip(self):    
+        assert_frame_equal(self.parq.to_df(), self.df)
+
+    def testColumns(self):
+        columns = ['coord_ra', 'coord_dec']
+        assert_frame_equal(self.parq.to_df(columns=columns), 
+                           self.df[columns])
+
+        # Quietly ignore nonsense columns
```

Comment:
Is silently ignoring bad inputs what we want, or is not ignoring them problematic somehow?